src/modules/users/users.controller.ts

```diff
@@ -48,6 +52,13 @@ export class UsersController {
 
         const newUser: User = await this.usersService.create(userDto);
 
+        const emailData = {
+          to: userDto.email,
+          templateId: EmailService.TEMPLATE_IDS.WELCOME_MESSAGE,
+        };
+        
+        this.emailService.send(emailData)
```

Comment:
Need to make sure that auth0 is not sending welcome email too

Reply:
Do you know if it is?  I am less familiar with which emails they send.

Reply:
@moshfeu I signed up with a new account and haven't seen a welcome email yet.  I *did* receive an email to verify account ownership, but nothing else.